### PR TITLE
WIP - Added three modules to the main Pantheon help document.

### PR DESCRIPTION
**Pull request by adahms**
Added the three remaining modules for our first release to the main Pantheon help document.

The only changes made are to change commented-out includes into actual modules.

### CODE REVIEW

**Comment by adahms** [comment]
@redhataccess/eud-tech-review - Please review for technical completeness and accuracy.

**Comment by benradey** [comment]
Tech review complete; I see the change here is just a few includes. I assume the included files have been previously reviewed.

**Comment by emarcusRH** [comment]
looks fine to me

**Comment by adahms** [comment]
Thanks, @emarcusRH!

I think we need a formal '+' in a review on the PR itself, so I've raised a review request for you here.

I'm sorry to ask, but could you approve the changes through the review interface? I'll push it on to final merge from there.

**Comment by adahms** [comment]
@redhataccess/eud-tech-review - Ready for merge.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ include::modules/ref_pantheon-user-interface.adoc[leveloffset=+2]
 
 include::modules/proc_log-in-to-pantheon.adoc[leveloffset=+3]
 
-// include::<pending>
+include::assembly-searching-modules.adoc[leveloffset=+2]
```
Comment (noopurmath3) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc:47]: I don't think we support nested assemblies in pantheon v2.
@xdavidson can you please confirm?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc [hunk 2]
```diff
@@ -28,9 +28,9 @@ include::modules/ref_prerequisites.adoc[leveloffset=+1]
 [id="{ProductShortName}-concepts"]
 == {ProductShortName} Concepts
 
-// include::modules/<pending>
+include::modules/con_module-metadata-in-pantheon.adoc[leveloffset=+2]
```
Comment (noopurmath3) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc:31]: Looks like this file `con_module-metadata-in-pantheon.adoc` also needs to be added to the PR, it is not present on the master branch in /modules

**Pull request merged**